weasyl/controllers/admin.py

```diff
@@ -79,7 +79,8 @@ def admincontrol_manageuser_post_(request):
                           catchphrase=form.catchphrase.strip() if form.ch_catchphrase else None,
                           birthday=form.birthday if form.ch_birthday else None,
                           gender=form.gender if form.ch_gender else None,
-                          country=form.country if form.ch_country else None)
+                          country=form.country if form.ch_country else None,
+                          remove_social=form.remove_social if form.remove_social else None)
```

Comment:
Just pass `form.remove_social` directly; it’s a list. The other ones have `if`s because they could be empty strings.